### PR TITLE
LRSD-103 [CP 2.0] Large File Uploader form page

### PR DESCRIPTION
**Pull request by pedroliveirav**
[LRSD-103](https://liferay.atlassian.net/browse/LRSD-103)

### CODE REVIEW

**Comment by pedroliveirav** [comment]
ci:test:sf

**Comment by kylembischof** [comment]
`attachtofile` is kind of a confusing route name. can we rename to be more clear that its the large file uploader?

**Comment by pedroliveirav** [comment]
> `attachtofile` is kind of a confusing route name. can we rename to be more clear that its the large file uploader?

@kylembischof  I've renamed the route and the main component to “Attachment File Uploader”, is that okay now?

File: workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/attachment-file-uploader/components/AttachmentUploader/index.jsx [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+import { Button as ClayButton } from '@clayui/core';
+
+const AttachmentUploader = () => {
+	const [fileAttached, setFileAttached] = useState(null);
+	const [isCheckboxChecked, setIsCheckboxChecked] = useState(false);
+	const [attachmentComment, setAttachmentComment] = useState('');
+	const [ticketAttachmentId, setTicketAttachmentId] = useState(null);
+	const [gcsSessionURL, setGcsSessionURL] = useState('');
+
+	const zendeskTicketId = 378777;
```
Comment (kylembischof) [inline workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/attachment-file-uploader/components/AttachmentUploader/index.jsx:16]: I think this `zendeskTicketId` shouldn't be hardcoded like this?

Also, can we remove the `zendesk` reference from these variables so we have less to change when/if we migrate to JSM? Can rename all these variables to just like `ticketId` so it stays relevant for both zendesk and jira?

**Review by kylembischof — COMMENTED** [review]
yeah that route naming is more clear, but I'll leave it to @amosfong to make the final call on naming conventions

File: workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/attachment-file-uploader/components/AttachmentUploader/index.jsx (hunk 1 above)
Comment (pedroliveirav) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/attachment-file-uploader/components/AttachmentUploader/index.jsx:16]: The “ticketId” variable will come from parameters in the URL, the behavior of the URL is in another ticket that hasn't been started yet, so I was instructed to mock the value.

About the name change, I'll rename it.

Comment (kylembischof) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/attachment-file-uploader/components/AttachmentUploader/index.jsx:16]: Thanks for the clarification, sounds good to mock for now until the other ticket is completed.

**Comment by amosfong** [comment]
hey pedro, do you if it's possible to use .js instead of .jsx extensions? there's quite a bit of formatting issues and sourceformatter skips jsx files. I'd like to start using .js if possible

**Comment by amosfong** [comment]
@pedroliveirav , I'm going to just do some minor formatting and merge this first. We can look into changing the extension later

**Comment by amosfong** [comment]
Made some changes here:
https://github.com/liferay-customer-solutions/liferay-portal/pull/276

-renamed to .js and then back so I could run yarn format. You can see the big difference
-renamed some variables

**Comment by pedroliveirav** [comment]
> Made some changes here: #276
> 
> -renamed to .js and then back so I could run yarn format. You can see the big difference -renamed some variables

Thanks @amosfong

**Comment by pedroliveirav** [comment]
#resent: https://github.com/liferay-customer-solutions/liferay-portal/pull/276

**Pull request closed without merging**